### PR TITLE
reHSLa and reRGBa ignore case

### PR DESCRIPTION
**Pull request by durga598**
ignore case for rgba and hsla, as if pass in uppercase it was returning default color

### CODE REVIEW

**Comment by asturur** [comment]
I think this is OK.

**Comment by asturur** [comment]
Do you mind adding a small test to the suite to test it?

like at file test/unit/color.js
dduplicate the test at line 176 making it with uppercase and do the same for an hsla test.

**Comment by durga598** [comment]
I added.

File: test/unit/color.js [hunk 1]
```diff
@@ -88,8 +88,8 @@
     var oColor = new fabric.Color('ffffffff');
     assert.ok(typeof oColor.toHexa === 'function');
     assert.equal(oColor.toHexa(), 'FFFFFFFF');
-    oColor.setSource([255,255,255,0.8]);
-    assert.equal(oColor.toHexa(), 'FFFFFFCC');
+    oColor.setSource([255,255,255,0.1]);
+    assert.equal(oColor.toHexa(), 'FFFFFF1A');
```
Comment (asturur) [inline test/unit/color.js]: can i ask you the reason of this change here? i remember this one was here because of weird decimals.

Comment (durga598) [reply test/unit/color.js]: It was in my forked version.

Comment (durga598) [reply test/unit/color.js]: changed that to original

**Pull request merged**